README.md

```diff
@@ -67,6 +73,26 @@ wget https://github.com/kelseyhightower/confd/releases/download/v0.16.0/confd-0.
 chmod +x /usr/local/bin/confd
 ```
 
+##### Basic installation instruction from sources
+
+```
+cd /opt
+sudo mkdir confd
+sudo chown $USER:$USER confd
```

Comment:
Why not to build it into `~/opt`? My personal opinion is to not spread the garbage anywhere outside user's home dir. Especially if we are not going to make these ... application part of the system.

Not mandatory because of #4521